Training/2.0 Libraries.md

```diff
@@ -33,16 +38,18 @@ Create a DataFrame using the Pandas library. Your DataFrame should represent the
 [Patient_table.pdf](https://github.com/SoFia2401/gubmes-health-data-analytics/files/7404366/Patient_table.pdf)
 
 
-```
-
+```py
+data = {"A":[50,"Arrythmia"],"B":[60,"Heart Failiure"],"C":[70,"Unstable Angina"]}
+dataFrame = pd.DataFrame(data=data , columns= ["Patient","Age","Heart Condition"])
```

Comment:
Good attempt, but unfortunately this will yield a table that looks like this:
**index   A   B   C**
0   50   60   70
1   Arrythmia   Heart Failure   Unstable Angina
The table is transposed so your column labels will in fact have to be row labels, which is not ideal for standard data sets.

The correct way to convert a dictionary of this form to a data frame would be using the pandas.DataFrame.from_dict() function. Have a look at the documentation on how you can use it here!

Alternatively, you can define your dictionary like this: 
`data = {'Patient':['A','B','C'], 'Age':[50,60,70], 'Heart condition':['Arrhythmia','Heart Failure', 'Unstable Angina']}`
and then use: `dataframe = pd.DataFrame(data)`

Ideally, you should learn both methods, because you will need both in future projects depending on the context.